src/index.js

```diff
@@ -48,4 +48,8 @@ var namespaces         = 0
 	return getter;
 }
 
-module.exports = { getNamespace, getCountNamespaces, startRequest, bind, patch };
+global.requestLocalStorage = global.requestLocalStorage || {};
+
+module.exports = global.requestLocalStorage[REQUEST_LOCAL_STORAGE_VERSION] || { getNamespace, getCountNamespaces, startRequest, bind, patch };
+
+if (!global.requestLocalStorage[REQUEST_LOCAL_STORAGE_VERSION]) global.requestLocalStorage[REQUEST_LOCAL_STORAGE_VERSION] = module.exports;
```

Comment:
Yeah, this looks safer.

The build-time replacement of `REQUEST_LOCAL_STORAGE_VERSION` is a neat trick, but the cost of just looking it up at run-time isn't really that great.  Might be a case where simplicity is worth it.  Either way eslint is choking on it, so you'll need to do _something_ to get ci passing. :)